pkgs/by-name/fi/firebase-tools/package.nix

```diff
@@ -20,7 +20,18 @@ buildNpmPackage {
   pname = "firebase-tools";
   inherit version src;
 
-  npmDepsHash = "sha256-3+XeXK3VGIs4Foi9iW9Kho/Y0JsTQZ7p+582MPgdH1A=";
+  npmDepsHash = "sha256-lQmoemIxzy2biu80UTR2eA+KJBeXWPh0xZRs/1of0eo=";
+
+  patches = [
+    # Use modern version of `ajv` instead of the four year old default in 13.29.1
+    (fetchpatch {
+      name = "ajv 6.2.6 -> 8.1.1";
```

Comment:
I don't think we should have a space in a derivation's name.

I was thinking of something like `bump-ajv.patch`

Reply:
Done.